x-pack/plugins/apm/public/components/app/Main/route_config/route_names.tsx

```diff
@@ -26,6 +26,6 @@ export enum RouteName {
   SERVICE_NODES = 'nodes',
   LINK_TO_TRACE = 'link_to_trace',
   CUSTOMIZE_UI = 'customize_ui',
-  RUM_OVERVIEW = 'rum_overview',
   ANOMALY_DETECTION = 'anomaly_detection',
+  CLIENT_SIDE_MONITORING = 'client-side-monitoring',
```

Comment:
let's use underscores here instead of dashes: `client_side_monitoring`. I realise there are some inconsistencies but I think we prefer underscores.